src/sections/target/Bibliography/index.js

```diff
@@ -0,0 +1,9 @@
+export const definition = {
+  id: 'bibliography',
+  name: 'Bibliography',
+  shortName: 'B',
+  hasData: () => true,
```

Comment:
Just double-checking that is the intended behaviour (we always have literature data)